### DISCUSSION TITLE
Is it possible to call a Pysript code from outside tool (or command line)?

### DISCUSSION THREAD

**Discussion by itispip** [custom-components/pyscript#214] [Ideas]
For example, I have a website in PHP, not the web portal of HA itself but the website server is in the site machine as HA.

Is that possible that user click a button the website, and PHP can execute a cmdline (eg. python xxx.py)  to trigger a pyscript automation code in HA?

**Comment by nashant** [comment]
You could create a webook which your button POSTs to and give the function an `@event_trigger` decorator which listens for that webhook event
